lib/Experian/IDAuth.pm

```diff
@@ -234,8 +225,7 @@ sub _build_person_tag {
             return;
         }
     } else {
-        warn "Invalid date of birth [$dob] for " . $self->{client_id};
-        return;
+        croak "Invalid date of birth [$dob] for " . $self->{client_id};
```

Comment:
changing warn to `croak` so not it will die of errors?

Reply:
I want the module to die of errors, so that error handling can be done by the module user.